plugins/modules/azure_rm_afdendpoint.py

```diff
@@ -0,0 +1,351 @@
+#!/usr/bin/python
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+#
+# Python SDK Reference: https://learn.microsoft.com/en-us/python/api/azure-mgmt-cdn/azure.mgmt.cdn.operations.afdendpointsoperations?view=azure-python
+#
+# TODO: Name check the URL
+# 
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: azure_rm_afdendpoint
+
+version_added: ""
```

Comment:
Missing version
```suggestion
version_added: ""
```

Reply:
Was not sure what version to assign it...putting in 2.5.0 unless I get further guidance.  Thanks!